pyspeckit/spectrum/models/ammonia15n.py

```diff
@@ -0,0 +1,1092 @@
+"""
+========================================
+15NH3 inversion transition TROT fitter
+========================================
+
+Ammonia inversion transition TROT fitter translated from Erik Rosolowsky's
+https://github.com/low-sky/nh3fit
+
+.. moduleauthor:: Adam Ginsburg <adam.g.ginsburg@gmail.com>
+modified for 15n-bearing isotopologue by:: Elena Redaelli <eredaelli@mpe.mpg.de>
+
+Module API
+^^^^^^^^^^
+
+#############TODO################
+Implement the equivalent of the cold_ammonia_model for 14NH3 in 15NH3
+to allow for Tkin to be fitted (instead of Trot)
+
+"""
+from __future__ import division
+
+import numpy as np
+from ...mpfit import mpfit
+from ...spectrum.parinfo import ParinfoList,Parinfo
+from . import fitter
+from . import model
+import matplotlib.cbook as mpcb
+import copy
+from astropy import log
+from six import iteritems
+from . import mpfit_messages
+import operator
+import string
+import warnings
+
+from .ammonia15n_constants import (line_names, freq_dict, aval_dict, ortho_dict,
+                                voff_lines_dict, tau_wts_dict)
+
+TCMB = 2.7315 # K
+
+def ammonia15n(xarr, trot=20, tex=None, ntot=12, width=1, xoff_v=0.0, fortho=0.0,
```

Comment:
Is there any difference between this function and the 14N version of the function?  If the code is all the same, but just the constants are changing, we should not duplicate all the code but should instead re-use it with different constants in place.

Reply:
You're indeed right, there is no difference. How do you suggest implementing the call either to 14nh3 or 15nh3 spectroscopy?

Reply:
We can just add an option `ammonia_constants=<either 15nh3 or 14nh3` to the existing ammonia model, then in your ammonia15n.py file, you would have, e.g.:

```python
def ammonia15n(*args, **kwargs):
     return ammonia(*args, ammonia_constants=ammonia15nconstants, **kwargs)
```
and that would be pretty much the whole code

We can talk more about how to do this next week maybe (after the JWST deadline)

Reply:
Sure. Good luck with your proposals meanwhile!